ui/raidboss/data/05-shb/trial/castrum-ex.js

```diff
@@ -0,0 +1,123 @@
+import ZoneId from '../../../../../resources/zone_id.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import Conditions from '../../../../../resources/conditions.js';
+import { Responses } from '../../../../../resources/responses.js';
+
+export default {
+  zoneId: ZoneId.CastrumMarinumExtreme,
+  triggers: [
+    {
+      id: 'CastrumEx Optimized Ultima',
```

Comment:
Sorry for this tiny nit, but can you name this file `emerald_weapon-ex` and ids `EmeraldEx` to match the way that Ruby Weapon looks?

Reply:
resolved